apps/client/src/pages/WrapperPage.js

```diff
@@ -110,27 +110,26 @@ const WrapperPage = ({ checker, topic, resetChecker }) => {
             <StepByStepItem
               checked={isFinished("questions")}
               customSize
-              done={isFinished("locationResult")}
+              done={isFinished("locationResult") || checker.stack.length > 0}
```

Comment:
`checker.stack.length > 0` is equal to `checker.stack.length` But why do we need this in the first place? We can only show questions when locationResult is finished right?